open_spiel/games/sheriff.cc

```diff
@@ -0,0 +1,392 @@
+// Copyright 2020 DeepMind Technologies Ltd.All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.#include
+// "third_party/open_spiel/games/battleship.h"
+
+#include "open_spiel/games/sheriff.h"
+
+#include <algorithm>
+
+#include "open_spiel/abseil-cpp/absl/strings/str_cat.h"
+
+namespace open_spiel {
+namespace sheriff {
+namespace {
+inline constexpr const Player kSMUGGLER = Player{0};
+inline constexpr const Player kSHERIFF = Player{1};
+
+template <typename T>
+void StrAppendVector(std::string* s, const std::vector<T>& v) {
+  absl::StrAppend(s, "[");
+  for (size_t index = 0; index < v.size(); ++index) {
+    if (index > 0) absl::StrAppend(s, ",");
+    absl::StrAppend(s, v[index]);
+  }
+  absl::StrAppend(s, "]");
+}
+}  // namespace
+
+SheriffState::SheriffState(
+    const std::shared_ptr<const SheriffGame> sheriff_game)
+    : State(sheriff_game), sheriff_game_(sheriff_game) {}
+
+Player SheriffState::CurrentPlayer() const {
+  if (!num_illegal_items_) {
+    // The smuggler still hasn't decided the number of illegal items to
+    // place in the cargo. The game has just begun.
+    return kSMUGGLER;
```

Comment:
Can you make the constants follow this format:

kSmuggler
kSheriff

(just not all caps)